.codesandbox/ci.json

```diff
@@ -1,4 +1,7 @@
 {
-    "sandboxes": ["new", "vanilla"],
```

Comment:
Please add this back to fix CodeSandbox CI.

Reply:
Will also be needed once we figure out how to get CodeSandbox links in Storybook :)